ml-prims/src/linalg/svd.h

```diff
@@ -156,7 +151,7 @@ void svdJacobi(math_t *in, int n_rows, int n_cols, math_t *sing_vals,
                math_t *left_sing_vecs, math_t *right_sing_vecs,
                bool gen_left_vec, bool gen_right_vec, math_t tol,
                int max_sweeps, cusolverDnHandle_t cusolverH, cudaStream_t stream,
-               DeviceAllocator &mgr) {
+               std::shared_ptr<deviceAllocator> allocator) {
```

Comment:
Could pass in cumlHandle_impl instead of cuBLAS handle, cuSolver handle and allocator.

Reply:
will be discussed in #498.